Makefile

```diff
@@ -18,6 +18,7 @@ prod: prod-image
 
 prod-test: prod-image
 	. environment/env-prod; docker run \
+		-t \
```

Comment:
Why do we need this now but not before? (it is pseudo tty correct?)

Reply:
Yes, it allows us to send `SIGINT` to the container and stop the test suite from running if we want to restart before everything finishing (nice since the suite can take up to 2 minutes to complete at times). Also gets us colored/formatted test output.